frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -178,6 +180,88 @@ export const surveyLogic = kea<surveyLogicType>([
                 editingSurvey: (_, { editing }) => editing,
             },
         ],
+        survey: [
+            { ...NEW_SURVEY } as NewSurvey | Survey,
+            {
+                setDefaultForQuestionType: (state, { type }) => {
+                    if (type === SurveyQuestionType.Open) {
```

Comment:
errr, it doesn't look like you pushed your recent changes to this?

Reply:
I had to resolve an eslint issue before committing. Pushed now!